styx-e2e-test/flyte-test-workflow/README.md

```diff
@@ -0,0 +1,57 @@
+---
+**Note**
+
+Everything has been deployed to flyte already. We only need to update the workflow unless it is really
+ necessary. Then you should follow the steps here.
+---
+
+#### Setup
+Install `flytekit` which will install: pyflyte, flyte-cli and flytekit_build_image.sh 
+```
+$ virtualenv .venv -p python3.6
+$ source .venv/bin/activate
+$ pip install -r requirements.txt
+```
+
+#### Build and push image
+Build and push image,
+
+```
+$ TAG=$(git log --format='%H' -n 1) REGISTRY=eu.gcr.io/styx-oss-test \
+ flytekit_build_image.sh .
+```
+
+To only build, do,
+```
+$ flytekit_build_image.sh .
+```
+
+#### Port forwarding to flyteadmin in the test cluster
+Access to flyteadmin is needed to deploy the workflow. You can create port forwarding to flyteadmin with this command
+```
+kubectl port-forward -n flyte port-forward deployment/flyteadmin 8089:8089
+```
+
+#### Deploy to flyte 
+Deploys workflows, tasks, launch plans and image to flyte.
+
+```
+docker run --network host \
+-e FLYTE_PLATFORM_URL='host.docker.internal:8089' \
+"eu.gcr.io/styx-oss-test/flyte-test-workflow:$(git log --format='%H' -n 1)" \
+pyflyte -p flytesnacks -d development -c sandbox.config register workflows
```

Comment:
does this flytesnacks have anything to do with https://github.com/lyft/flytesnacks ?

Reply:
Not related. It's just sandbox deployment seeding a few projects and `flytesnacks` is one of those.